src/Altinn.Platform/Altinn.Platform.Storage/Storage.Interface/Models/OnEntryConfig.cs

```diff
@@ -0,0 +1,19 @@
+using Newtonsoft.Json;
+
+namespace Altinn.Platform.Storage.Interface.Models
+{
+    /// <summary>
+    /// The on entry configuration
+    /// </summary>
+    public class OnEntryConfig
+    {
+        /// <summary>
+        /// Defines what should be shown on entry.
+        /// </summary>
+        /// <remarks>
+        /// Valid selections include: new-instance, startPage, and a string mathcing the layoutSetId
```

Comment:
Minor typo
```suggestion
        /// Valid selections include: new-instance, startPage, and a string matching the layoutSetId
```

Reply:
Btw, new-instance and startPage is for a later story and won't do anything just yet - so maybe remove them for now?